tox.ini

```diff
@@ -8,7 +8,7 @@ deps = flake8==3.6.0
 setenv = PYFLAKES_ERROR_UNKNOWN=1
 commands =
     python --version --version
-    python -m unittest discover pyflakes
+    python -m unittest discover pyflakes {posargs}
```

Comment:
This seems orthogonal

Reply:
I moved this to https://github.com/PyCQA/pyflakes/pull/624